app/controllers/order/client_information.js

```diff
@@ -47,102 +50,124 @@ export default Ember.Controller.extend(cancelOrder, {
     ];
   }),
 
-  beneficiaryParams(){
-    let title = Ember.$("select#title option").toArray().filter((title) => title.selected === true)[0].value;
+  beneficiaryParams() {
+    let title = Ember.$("select#title option")
+      .toArray()
+      .filter(title => title.selected === true)[0].value;
     var beneficieryParams = {
-      first_name: this.get('firstName'),
-      last_name: this.get('lastName'),
+      first_name: this.get("firstName"),
+      last_name: this.get("lastName"),
       title: title,
-      identity_number: this.get('identityNumber'),
-      phone_number: this.get('mobile'),
-      order_id: this.get('order.id'),
-      identity_type_id: this.identityTypeId(),
+      identity_number: this.get("identityNumber"),
+      phone_number: this.get("mobile"),
+      order_id: this.get("order.id"),
+      identity_type_id: this.identityTypeId()
     };
     return beneficieryParams;
   },
 
-  identityTypeId(){
-    return this.get('selectedId') === 'hkId' ? 1 : 2;
+  identityTypeId() {
+    return this.get("selectedId") === "hkId" ? 1 : 2;
   },
 
   actionType(isOrganisation, beneficiaryId) {
     let actionType;
     if (isOrganisation && beneficiaryId) {
-      actionType = 'DELETE';
+      actionType = "DELETE";
     } else if (!isOrganisation && beneficiaryId) {
-      actionType = 'PUT';
+      actionType = "PUT";
     } else if (!isOrganisation && !beneficiaryId) {
-      actionType = 'POST';
+      actionType = "POST";
     } else {
       actionType = null;
     }
     return actionType;
   },
 
+  editOrder: task(function*(orderParams, isOrganisation) {
```

Comment:
Why use task and not async/await?

Reply:
I had used async/await but tests are failing because of code coverage plugin.